internal/it/util.go

```diff
@@ -458,3 +458,37 @@ func WaitEventuallyWithTimeout(t *testing.T, wg *sync.WaitGroup, timeout time.Du
 		t.FailNow()
 	}
 }
+
+// Cloud APIs
+
+func LoginToHazelcastCloudUsingEnvironment() error {
+	return rc.LoginToHazelcastCloudUsingEnvironment(context.Background())
```

Comment:
Could you add a `ctx context.Context` parameter to these APIs? Like, this one would become:
```
func LoginToHazelcastCloudUsingEnvironment(ctx context.Context) error {
	return rc.LoginToHazelcastCloudUsingEnvironment(ctx)
}
```

Reply:
Updated